project.php

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project Manager - Project Name</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="h-full">
+<header class="p-3 bg-teal-50 flex justify-between">
+    <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
+    <div class="pr-3 flex">
+        <a href="project.php" class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+    </div>
+</header>
+<main class="p-3">
+
+    <?php
```

Comment:
Same as my comment in task, this block of php should go at the very top of the file to make it quicker to see which other files are being used